index.html

```diff
@@ -53,7 +53,7 @@
   </div>
   <div class="item-menu"><h3 onclick="openMain()"href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></h3></div>
       <div class="hello"><p>"Hello! Sing for peace"</p></div>
-      <p id="jedan">EXIT Music Festival Novi Sad 2022</p>
+      <h1 id="jedan">EXIT Music Festival Novi Sad 2022</h1>
       <p id="h1"class="con">Bringing together through culture and providing a space for
         celebration where people from all over the world can share their
         cultures.</p>
```

Comment:
- [ **OPTIONAL** ] Good job applying the change required by the previous reviewer. But I think it would be better if you apply this same approach in your `about.html` page, on the same text for good semantics reasons.